src/components/BorderedBox/index.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from '@emotion/styled'
+
+const BorderedBox = styled.div`
+  padding: ${({ theme }) => theme.space['3']};
+  border-radius: ${({ theme }) => theme.radii.default};
+  border: 1px solid ${({ theme }) => theme.colorsDeprecated.gray350};
```

Comment:
Can we also migrate to the new colors since this is a new component?

Reply:
done thanks :)